src/main/ruby/codacy/brakeman/DocumentationGenerator.rb

```diff
@@ -83,7 +83,9 @@
           code == :CVE_2015_7578 ||
           code == :CVE_2015_7580 ||
           code == :CVE_2016_6317 ||
-          code == :CVE_2018_3741
+          code == :CVE_2018_3741 ||
+          code == :custom_check
```

Comment:
Brakeman added support for custom checks. But we can't generate docs for that.
https://github.com/presidentbeef/brakeman/releases/tag/v4.6.0